### PR TITLE
Refine CTA button guidance

### LINKED ISSUE #2961: Button – Refine CTA button vs primary action button guidance
Feedback has been provided by Edgar that the current guidance is not clear enough that "CTA" is for marketing/value prop use cases. For the cases where "enabling a new feature" etc. might be appropriate in a product as a CTA, we should clearly bullet those use cases.

Content in question: https://design-system.hpe.design/components/button?q=call#primary-buttons

- [ ] Touch base with a few other stakeholders to get their impressions of this content
- [ ] Refine guidance accordingly
- [ ] Include Edgar as a reviewer as he initially raised the feedback

### PR DESCRIPTION
**Pull request by taysea**
[Deploy Preview](https://deploy-preview-2984--keen-mayer-a86c8b.netlify.app/components/button#when-to-use-a-cta-button)

#### What does this PR do?

Have refined the CTA button guidance to more clearly articulate some of the use cases and reasoning of when a CTA button may be appropriate.

#### What testing has been done on this PR?

In addition to the feature you are implementing, have you checked the following:

**General UX Checks**
- [ ] Small, medium, and large screen sizes
- [ ] Cross-browsers (FireFox, Chrome, and Safari)
- [ ] Light & dark modes
- [ ] All hyperlinks route properly

**Accessibility Checks**
- [ ] Keyboard interactions
- [ ] Screen reader experience
- [ ] Run WAVE accessibility plugin (Chrome)

**Code Quality Checks**
- [ ] Console is free of warnings and errors
- [ ] Passes E2E commit checks
- [ ] Visual snapshots are reasonable

#### What are the relevant issues?
Closes #2961

### CODE REVIEW

File: aries-site/src/pages/components/button.mdx [hunk 1]
```diff
@@ -144,7 +144,20 @@ Primary buttons have increased visual weight through placement and contrast, dra
 
 Call-to-action (CTA) buttons entice a user down a curated path leading to incremental value. A primary call-to-action is meant to encourage the most desired action you want visitors to take, leading prospects and customers towards conversion, whether that be to learn more, register, test-drive and even buy a product.
 
-Specifically, CTA Buttons should be used whenever a prospect or customer is unlocking new value from HPE’s edge-to-cloud promise. This primarily occurs in marketing contexts, but can also occur within an enterprise application where a service, feature, trial opportunity, or event is being promoted.
+Specifically, CTA Buttons should be used whenever a prospect or customer is unlocking new value from HPE’s edge-to-cloud promise. This occurs in marketing contexts such as:
```
[the remark this answers is not in the document]
Comment (taysea) [reply aries-site/src/pages/components/button.mdx:147]: @halocline any thoughts on how to reword this?

[the remark this answers is not in the document]
Comment (halocline) [reply aries-site/src/pages/components/button.mdx:147]: Any better?

"Specifically, CTA buttons should be used when an HPE customer will gain value from HPE's edge-to-cloud promise."

synonyms for gain: realize, extract, get

[the remark this answers is not in the document]
Comment (taysea) [reply aries-site/src/pages/components/button.mdx:147]: OR:

"Specifically, CTA buttons should be used when a potential or current HPE customer will gain value from HPE's edge-to-cloud promise."

I liked the note about "prospect" in the initial sentence. @britt6612 thoughts?

[the remark this answers is not in the document]
Comment (halocline) [reply aries-site/src/pages/components/button.mdx:147]: prospective? 

potential works too. Just trying things.

[the remark this answers is not in the document]
Comment (taysea) [reply aries-site/src/pages/components/button.mdx]: Done! Went with prospective.

**Review by halocline — APPROVED** [review]

**Comment by taysea** [comment]
Got approval from Edgar via slack, merging.

**Pull request merged**